src/test/java/io/xstefank/wildfly/bot/PRRuleTitleBodyCheckTest.java

```diff
@@ -122,4 +124,77 @@ void testFailedTitleBodyCheck() throws IOException {
                     Util.verifyFormatFailure(repo, gitHubJson, "title");
                 });
     }
+
+    @Test
+    void testTitleBodyCheckTitleRegex() throws IOException {
+        GitHubJson gitHubJson = GitHubJson.builder(VALID_PR_TEMPLATE_JSON)
+                .title("[WFLY-00000] Metrics PR")
+                .action(Action.OPENED)
+                .build();
+        wildflyConfigFile = """
+                wildfly:
+                  rules:
+                    - titleBody: "metrics|micrometer"
+                      notify: [7125767235]
+                  format:
+                    commit:
+                      enabled: false
```

Comment:
Is there a reason for disabling commit format checks? IIRC the #101 stated that checks should be enabled if we don't specifically want them disabled